### PR TITLE
Wechaty 1.0

### PR DESCRIPTION
**Pull request by hcfw007**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,15 +60,14 @@
...
-    "wechaty-puppet": "^0.41.9"
-  },
   "dependencies": {
+    "file-box": "^1.4.14",
     "flash-store": "^0.12.7",
     "fs-extra": "^8.1.0",
     "gerror": "^1.0.16",
     "rxjs": "^7.5.1",
-    "wechaty-puppet": "^0.41.9",
+    "wechaty-puppet": "^1.10.2",
```
Comment (huan) [inline package.json:69]: v1.11 is recommended because it has new breaking changes.

Comment (hcfw007) [reply package.json]: OK. I used the default latest tag when installing.

**Comment by hcfw007** [comment]
So far my wechaty-puppet-1.0-migration package only provide esm dist. I'll make it available on cjs asap.

**Review by su-chang — APPROVED** [review]

**Comment by hcfw007** [comment]
@su-chang ping. If you don't review it, I might have to solve another conflict merge...

**Review by su-chang — APPROVED** [review]

**Pull request merged**